src/server/pkg/sync/sync_v2.go

```diff
@@ -0,0 +1,17 @@
+package sync
```

Comment:
This is not a great name for a package (because of the conflict the the standard library). This might be a good opportunity to create a `pfssync` package, especially since the new codepath appears totally separate.

Reply:
Agree, sync has always been a bad name for this package and it's been an annoyance for a while.

Reply:
Yeah, I'm on board with making a change here. We might be better off just changing the path to something under `/src/server/pfs` if we intend to keep the name `sync` (we would still use the `pfssync` import rename). I'm fine with either, or an alternative name, so just tell me what makes sense to you guys and I will change it.

Reply:
I think just renaming to `pfssync` probably makes the most sense. That way we don't have to worry about importing it correctly.